frontend/src/tests/instructor-permission-test.js

```diff
@@ -46,6 +46,63 @@ export function instructorsPermissionTests(api) {
         expect(respSwitchBackUser).toHaveStatus("success");
     }
 
+    /**
+     * Updates a position to include instructor with <instructorId> and
+     * creates a DDAH for one of the assignments realted to that
+     * instructor.
+     *
+     * @param instructorId: int - the unique Id of an instructor
+     *
+     * @returns {Promise<Ddah>}
+     */
+    async function createDDAH(instructorId) {
+        // We first need to update position to include our instructor
+        const existingPosition = databaseSeeder.seededData.positions[0];
+        const updatedPosition = {
+            ...existingPosition,
+            instructor_ids: [...existingPosition.instructor_ids, instructorId],
+        };
+        const positionResponse = await apiPOST(
+            `/admin/positions`,
+            updatedPosition
+        );
+        expect(positionResponse).toHaveStatus("success");
+
+        // We then proceed to create a DDAH for that position
+        // Switch to instructor user so we only have assignments for that instructor
+        await switchToInstructorOnlyUser();
+        const assignments = await apiGET(
+            `/instructor/sessions/${session.id}/assignments`
+        );
+        expect(assignments).toHaveStatus("success");
+        expect(assignments.payload.length).toBeGreaterThan(0);
+        const newDdah = {
+            assignment_id: assignments.payload[0].id,
+            duties: [
```

Comment:
We could alternatively pass this as a parameter, but I opted for hard-coding it here, so that the test itself is easier to read/understand

Reply:
Hard coding is fine. We'll change it if we need more functionality.